src/main/java/xyz/nucleoid/plasmid/api/game/GameType.java

```diff
@@ -21,13 +22,17 @@
  * @see GameConfig
  */
 public final class GameType<C> {
-    public static final TinyRegistry<GameType<?>> REGISTRY = TinyRegistry.create();
+    /**
+     * @deprecated Use {@link PlasmidRegistries#GAME_TYPE} instead.
+     */
+    @Deprecated
+    public static final Registry<GameType<?>> REGISTRY = PlasmidRegistries.GAME_TYPE;
```

Comment:
This isn't binary compatible! It might be better to create stub TinyRegistry variant that just wraps the real one

Reply:
Ah yes, that went over my head. new solution should fix this